app/src/main/res/values-de/strings.xml

```diff
@@ -123,6 +133,7 @@
     <string name="organisations">Organisationsbaum</string>
     <string name="mvv">MVV</string>
     <string name="news">News</string>
+    <string name="kino">TU Kino</string>
```

Comment:
I think you are right.
However, I just stick to the status quo. It's the same with e.g. mvv